src/AcceptanceTests/ConfigureEndpointAzureServiceBusTransport.cs

```diff
@@ -72,7 +72,10 @@ public Task Configure(string endpointName, EndpointConfiguration configuration,
 
         // w/o retries ASB will move attempted messages to the error queue right away, which will cause false failure.
         // ScenarioRunner.PerformScenarios() verifies by default no messages are moved into error queue. If it finds any, it fails the test.
-        configuration.Recoverability().Immediate(retriesSettings => retriesSettings.NumberOfRetries(3));
+        var recoverability = configuration.Recoverability();
+        recoverability.Immediate(retriesSettings => retriesSettings.NumberOfRetries(3));
+
+        recoverability.DisableLegacyRetriesSatellite();
```

Comment:
This change is not strictly necessary but I guess it is still good

Reply:
Indeed. It helps to troubleshoot when there are no extra entities created.